snap/snapcraft.yaml

```diff
@@ -24,9 +24,19 @@ platforms:
     build-on: [arm64]
     build-for: [arm64]
 
+slots:
+  dma-latency-dev:
+    interface: custom-device
+    custom-device: cpu-dma-latency
+    devices:
+      - /dev/cpu_dma_latency
+
 plugs:
   process-control:
     interface: process-control
+  cpu-latency-dev:
+    interface: custom-device
+    custom-device: cpu-dma-latency
```

Comment:
Can't this be done with a system-files interface? I assume the application need access just to that particular file, for manipulating the CPU's C state. The [custom device interface](https://snapcraft.io/docs/custom-device-interface) is to create a specific interface for a device class.

Reply:
It would be possible, but unfortunately isn't allowed since [system-files shouldn't/cannot be used to access /dev/* devices](https://forum.snapcraft.io/t/system-files-interface-for-dev-gpiochip/38629/3?u=locnnil).